cmd/clusterawsadm/api/bootstrap/v1beta1/defaults.go

```diff
@@ -31,6 +31,8 @@ const (
 	DefaultStackName = "cluster-api-provider-aws-sigs-k8s-io"
 	// DefaultPartitionName is the default security partition for AWS ARNs.
 	DefaultPartitionName = "aws"
+	// DefaultPartitionNameUSGov is the default security partition for AWS ARNs.
+	DefaultPartitionNameUSGov = "aws-us-gov"
```

Comment:
```suggestion
	PartitionNameUSGov = "aws-us-gov"
```

Reply:
We can remove Default from DefaultPartitionNameUSGov

Reply:
done